tests/test_cli_download_files.py

```diff
@@ -94,7 +94,7 @@ def test_download_files_https_pycurl():
         os.remove(test_file)
     test_download_files = CliRunner()
     test_result = test_download_files.invoke(
-        download_files, ["--recid", 3005, "--protocol", "https"]
+        download_files, ["--recid", 3005, "--server", "https://opendata-dev.cern.ch"]
```

Comment:
This would be the only use of `opendata-dev.cern.ch`, so I think it is better to replace this with `SERVER_HTTPS_URI` from `config` so that we are using the same server consistently everywhere.